getting-started/authenticating-users/main_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestIndex(t *testing.T) {
+	tests := []struct {
+		path           string
+		wantStatusCode int
+		wantBody       string
+	}{
+		{
+			path:           "/",
+			wantStatusCode: http.StatusOK,
+			wantBody:       "Hello None\n",
+		},
+		{
+			path:           "/hello",
+			wantStatusCode: http.StatusNotFound,
+			wantBody:       "404 page not found\n",
+		},
+	}
+
+	for _, test := range tests {
```

Comment:
subtest?

Reply:
They're very fast, so I don't think subtests are necessary. But, I added the path to the failure messages to make them more helpful.